docs/configuration/index.md

```diff
@@ -749,6 +749,30 @@ We provide the following variables:
 (*string*)  Specifies the SIG (special interest group). This option is used
 during cloning of the dist-git repos that are maintained in the SIG namespaces.
 
+#### sync_test_job_statuses_with_builds
+
+(*bool*) When set to `true` (the default), test job statuses are updated together with their corresponding
+build job statuses. When set to `false`, while test jobs are waiting for their corresponding build jobs
+to finish, their statuses remain in pending state and only build job statuses are updated, for example with
+`SRPM build is in progress...` or `Starting RPM build...`.
```

Comment:
Shouldn't we write down when an user needs to put this key to `false`. I think it is not straightforward guessing it. I see you just moved this doc...
Still I will add something like the following...
```suggestion
`SRPM build is in progress...` or `Starting RPM build...`.
You may want to disable the statuses update when you are experiencing problems with the number of Github API calls in your organization. This will lower the calls made by the Packit app.
```

Reply:
too late 😅

Reply:
@majamassarini We can extend the description later if we see that it really has an effect and how big the effect is.

Reply:
@majamassarini yes that's a good point, for context it would be nice to mention it